.github/workflows/build.yaml

```diff
@@ -103,7 +103,7 @@ jobs:
         name: Login into Docker Hub
         uses: docker/login-action@v1
```

Comment:
If we're changing things for our repo here anyway, should we also use our vendored version of the docker login plugin? See https://github.com/lightninglabs/lightning-terminal/blob/master/.github/workflows/docker.yml#L28 for example.

Reply:
Yeah, I included all of our Github Actions customizations in the final commit, so we can easily replay it on top of the vendor branch.